src/Entity/SocialAuth.php

```diff
@@ -30,6 +31,40 @@ class SocialAuth extends ContentEntityBase implements ContentEntityInterface {
   /**
    * Creating fields.
    */
+
+  /**
+   * {@inheritdoc}
```

Comment:
Please, put setters and getters of the same data field one above the other.
For instance, after setToken, put getToken.